mux.c

```diff
@@ -210,14 +215,35 @@ static int do_subneg(struct ios_ops *ios, unsigned char *buf, int len)
 static int logfd = -1;
 char *answerback;
 
-static void write_receive_buf(const unsigned char *buf, int len)
-{
-	if (!len)
-		return;
+static void write_receive_buf(const unsigned char *buf, int len) {
+    if (!len) return;
+	if (timestamps) {
```

Comment:
Indentation looks broken. Please fix.

Reply:
I used ClangFormat from VIM, and its automatically mess up this file for some reason, i will manually reformat and commit. 

Thanks.